{"version":1,"ops":[{"type":1,"author":{"id":"c750eb0960e029df836e56dc77bb8c148fcf5723"},"timestamp":1293237269,"metadata":{"github-id":"MDU6SXNzdWUyODQ4Njg0","github-url":"https://github.com/parrot/parrot/issues/476","origin":"github"},"title":"tailcall method invocation pops exception handlers","message":"Tailcalling a function call leaves the exception handlers created for the current frame active. Tailcalling a method invocation does not.\n\nOriginally http://trac.parrot.org/parrot/ticket/1903","files":null}]}